### ISSUE TITLE
[Bug]: Handle chunk load error better

### ISSUE THREAD

**Issue by SatishGandham** [appsmithorg/appsmith#24296]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

If Appsmith is upgraded to v2 from v1 while some clients are connected to us via v1

When clients connected via v1 try to navigate to a  different path, say from view to edit mode, v1 code will request files from v1 bundle [These are not loaded yet because of lazy loading] and these are not longer available on the server as it upgraded to v2.

This results in chunk load error.

Currently, we show a toast message saying "There is a new version, please refresh", may be change the message or find a way to solve it better.

### Steps To Reproduce

NA

### Environment

Production

### Version

Self Hosted

**Comment by somangshu** [comment]
@hetunandu suggested a easy fix on this by converting the `toast message` into a `non dismissable alert`, where the only action available is to refresh the page or refresh the page with keyboard shortcuts. No other action by the user must be allowed at this point in time.  

cc @SatishGandham @ginilpg

**Comment by somangshu** [comment]
Found the original context in #685. I see @nayan-rafiq is the contributor here!

@nayan-rafiq can you please verify if the above makes sense and help us take this forward. 

cc @ginilpg @sharat87

**Comment by ginilpg** [comment]
@somangshu We are working on this. Seems like this has been stopped after CDN integration. We will be adding more analytics to understand whether users are still experiencing this problem. Based on that we will follow up this. 

https://github.com/appsmithorg/appsmith/issues/22905

**Comment by somangshu** [comment]
Yeah I observed that too @ginilpg, closed #23883 on the same lines. 

Also, who should be assigned to this, and what are some expected timelines to track this on?

**Comment by SatishGandham** [comment]
Can we merge 22905 and 24296
This issue will still be present for self hosted instances
https://github.com/appsmithorg/appsmith/issues/22905#issuecomment-1617464671

**Comment by somangshu** [comment]
@SatishGandham @ginilpg can we close all issues and track progress in one place?
